apps/pie-docs/README.md

```diff
@@ -59,16 +59,23 @@ Our primary concerns are:
 4. Our shortcodes return the expected HTML
 
 ### Page testing
+
 We have route navigation tests that ensure all existing pages can be correctly navigated to. When you add new pages, these tests will fail as there are new unexpected pages (this is by design).
 
-In order to fix the tests, you will need to register the routes to your newly added pages by running from the root of the monorepo: `yarn test:generate-routes`. This will update the JSON file named `expected-routes.snapshot.json` with the URLs to your new page.
+Route tests need a production build so that they do not fail with `draft` page routes. For this, we recommend to:
+- stop any local instance of the doc site 
+- delete your `dist` folder 
+- create a production build with `yarn build --filter=pie-docs` 
 
+Once this is done, you can run route tests with `yarn test --filter=pie-docs`.
 From here, we run navigation, accessibility and visual tests against each route.
 
-#### Running the route tests
-Route tests are ran as part of `yarn test`.
+#### Adding new routes
+
+In order to fix the tests, you will need to register the routes to your newly added pages by running the following command from the root of the monorepo: `yarn test:generate-routes`. This will update the JSON file named `expected-routes.snapshot.json` with the URLs to your new page.
```

Comment:
```suggestion
In order to fix the tests, you will need to register the routes to your newly added pages by running the following command from the root of the monorepo: `yarn test:generate-routes`. This will update the JSON file named `expected-routes.snapshot.json` with the paths to your new pages.
```